src/sc.c

```diff
@@ -1467,4 +1465,19 @@ sc_version_minor (void)
   return SC_VERSION_MINOR;
 }
 
+/* Define helper macros temporally to convert the point version */
+#define _SC_TOSTRING(x) #x
+#define SC_TOSTRING(x) _SC_TOSTRING(x)
+
+int
+sc_version_point (void)
+{
+  /* SC_VERSION_POINT may contain a dot and/or dash,
+     followed by additional information */
+  return atoi (SC_TOSTRING (SC_VERSION_POINT));
+}
+
+#undef TOSTRING
+#undef _TOSTRING
```

Comment:
I didn't want to clutter the namespace with macros that are not used elsewhere. Hence, I `#undef`ed the macros above after their use in `sc_version_point`. Since you changed the name to include `SC_`, the `#undef` statements should probably be updated, too. What do you suggest, @cburstedde?

Reply:
Uh yes you're right on both counts.

Undefined behavior is generally not acceptable. We get around it this time since we have a test program. Let's see what happens if we don't have a point, say checking out v2.2. Will the test go through?

I didn't see the undef of TOSTRING. I'd tend to remove it. Still we may move the macro definitions to sc.h.

Reply:
Okay. Shall I do some of these steps or is it just easier for you to do that?